mathics/builtin/assignment.py

```diff
@@ -792,10 +800,130 @@ def rhs(expr):
 
     def format_definition_input(self, symbol, evaluation):
         'InputForm: Definition[symbol_]'
-
         return self.format_definition(symbol, evaluation, grid=False)
 
 
+
+
+
+
+
+def information_interpret_doc_string(ds):
+    # TODO: work a little bit the interpretation
+    ds = ds.split("\n")
+    out = ""
+    for s in ds:
+        if s[0:6]=="    >>":
+            # If we reach the test section, stop the loop.
+            break
+        out = out + "\n" + s
+    return out
```

Comment:
This is not really a robust way to parse the docstrings. I think it's better to reuse the existing stuff in `mathics/doc`.